scanpy/preprocessing/_simple.py

```diff
@@ -746,7 +746,7 @@ def _regress_out_chunk(data):
 
 
 @renamed_arg("X", "data", pos_0=True)
-@old_positionals("zero_center", "max_value", "copy", "layer", "obsm", "mask")
```

Comment:
Just to be clear, was `mask` ever needed in `old_positionals` since we never released it as a positional?